runtime/ftplugin/query.lua

```diff
@@ -2,5 +2,21 @@
 -- Language:	Tree-sitter query
 -- Last Change:	2022 Mar 29
 
+if vim.b.did_ftplugin == 1 then
+  return
+end
```

Comment:
```suggestion
end
vim.b.did_ftplugin = 1
```

Reply:
but wouldn't that prevent the lisp ftplugin from loading? This is why I not added it. The lisp ftplugin should set that variable